DeUrgenta/Features/CreateGroup/View/ContactsView.swift

```diff
@@ -0,0 +1,76 @@
+import SwiftUI
+
+protocol ContactsViewDelegate {
+    func contactsViewDidTapContinueButton(_ view: ContactsView)
+}
+
+struct ContactsView: View {
+    @ObservedObject var viewModel: CreateGroupViewModel
+    @State var searchText: String = ""
+
+    var delegate: ContactRowDelegate
+    var contactDelegate: ContactsViewDelegate
+
+    var filtredContacts: [ContactInfo] {
+        var tempArray: [ContactInfo] = viewModel.contacts
```

Comment:
In Swift, we should use specific names. Let's describe what this tempArray actually is.